config/jobs/kubernetes-sigs/kube-storage-version-migrator/ci.yaml

```diff
@@ -0,0 +1,31 @@
+presubmits:
```

Comment:
The e2e tests are not checked in yet, so I want to make the test a presubmit for now, so that I can debug my e2e test.

Reply:
You should probably make this always_run: false until then

Reply:
so far I'm the only contributor in that repo, so it doesn't really matter ;)